workflows/phyloplace.nf

```diff
@@ -15,35 +15,38 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 
 // Check mandatory parameters
 if (params.input) { 
-    //ch_input = Channel.fromPath(params.input) 
     Channel.fromPath(params.input)
         .splitCsv(header: true)
         .map { 
             [ 
                 meta: [ id: it.sample ], 
                 data: [ 
-                    queryfile:    file(it.queryfile),
-                    refalignment: file(it.refalignment),
+                    alignmethod:  it.alignmethod ? it.alignmethod    : 'hmmer',
+                    queryseqfile: file(it.queryseqfile),
+                    refseqfile:   file(it.refseqfile),
+                    hmmfile:      it.hmmfile     ? file(it.hmmfile)  : [],
```

Comment:
```suggestion
                    hmmfile:      it.hmmfile     ? file(it.hmmfile, checkIfExists: true)  : [],
```

This will cause the pipeline to fail if the file doesn't exist (instead of running until the point it needs the file and giving an error at that point. You can also add all files given through parameters to the list on line 13 in `workflows/phyloplace.nf` to get them checked on line 14 :)